.github/workflows/_reusable_pr-comment-list-changes.yml

```diff
@@ -0,0 +1,45 @@
+# WARN: this workflow may be reused in other workflows
+# * triggered by a pull_request_target (to work with fork repository). This workflow doesn't do any unsecure action regarding the pull_request_target context.
+# * called with secret: inherit
+# * with:
+#    permissions:
+#     pull-requests: write
+name: Comments PR with useful links
+
+on:
+  workflow_call: # this allows the workflow to be reused
+    inputs:
+      component-name:
+        type: string
+        description: "The name of the component to build."
+        required: false
+        default: "bonita"
+      pattern:
+        type: string
+        description: ""
+        required: false
+        default: "modules/**/pages/**/*.adoc"
+      # needed by content repositories (default master) and by the doc-site repository (computed)
+      doc-site-branch:
+        type: string
+        description: "The branch of the `bonita-documentation-site` used to download js files"
+        required: false
+        default: "master"
+
+jobs:
+  # IMPORTANT: the job id must be the same as in "_reusable_surge-deploy-preview.yml" as the surge-preview-tools action uses it to generate the preview URL (current limitation of v3.2.0)
```

Comment:
**nitpick:** add a comment in _reusable_surge-deploy-preview.yml to warn about the need for job.id synchronization as done here